third_party/library.sh

```diff
@@ -123,11 +123,36 @@ function generate() {
   generate_manifests "$path" "$@"
 }
 
+function run_yq() {
+  GO111MODULE=on run_go_tool github.com/mikefarah/yq/v4 yq "$@"
+}
+
 function install_yaml() {
-  if grep -q "CustomResourceDefinition" "$1"; then
-    kubectl apply --selector="knative.dev/crd-install=true" -f "$1"
-    kubectl wait --for=condition=Established --all crd
-  fi
+  local order=(
```

Comment:
this is cool. feels like there should be a way in kubectl to specify these sorts of orderings... but i guess that wouldn't be declarative

Reply:
I think kubectl just passes things along as they appear in `yaml` - in serving we structure our config directory so the resulting yaml is ordered in such a way so it works most of the time.

The `istioctl` output is somewhat ordered but